cloud/linode/client/client.go

```diff
@@ -11,7 +11,12 @@ import (
 type Client interface {
 	GetInstance(context.Context, int) (*linodego.Instance, error)
 	ListInstances(context.Context, *linodego.ListOptions) ([]linodego.Instance, error)
+	CreateInstance(ctx context.Context, opts linodego.InstanceCreateOptions) (*linodego.Instance, error)
 	GetInstanceIPAddresses(context.Context, int) (*linodego.InstanceIPAddressResponse, error)
+	AddInstanceIPAddress(ctx context.Context, linodeID int, public bool) (*linodego.InstanceIP, error)
+	DeleteInstanceIPAddress(ctx context.Context, linodeID int, ipAddress string) error
+
+	ShareIPAddresses(ctx context.Context, opts linodego.IPAddressesShareOptions) error
```

Comment:
nit: we can remove parameter names and just keep parameter types to keep it consistent with other lines in file. There are some lines which still have both parameter name and types, but in general I see folks trying to just keep parameter types.

Reply:
If we want to keep it consistent, I would prefer adding parameter names for clarity

Reply:
`Make it consistent! By converging on my standard...` 

I love it :D